docs/src/pages/landing-page/Index.vue

```diff
@@ -57,16 +57,24 @@
         Why should your team choose quasar?
       </div>
 
-      <div class="why-quasar-grid">
-        <template v-for="({icon, title, body, btnLabel, btnLink}, i) in whyQuasar" :key="i">
-          <why-quasar-card
-            :icon="icon"
-            :title="title"
-            :body="body"
-            :btn-label="btnLabel"
-            :btn-link="btnLink"
-          />
-        </template>
+      <div class="row justify-center">
```

Comment:
Ok, this confuses me (eg. there are infinite divs, one of those is both `row` and `column`, etc), let's check this out together when we meet with Nick